pytask_io/actions.py

```diff
@@ -0,0 +1,7 @@
+from enum import Enum
+
+
+class QueueActions(Enum):
+    START = 1
+    STOP = 2
+    IDLE = 3
```

Comment:
we can now issue queue actions , for example on run can do set up for tasks etc once. For the idle action type lets us know we have the option of stopping and closing the event loop and stop obviously does what it says.